### PR TITLE
[LC-754] remove add_peer from channel_infos

### PR DESCRIPTION
**Pull request by laurenceyoon**
- In order to get initial rep list, the `crep_root_hash` should be in its configuration.
- `channel_infos` will contain only the `CHANNEL_OPTION` configuration, not peer list.

### CODE REVIEW

File: loopchain/channel/channel_service.py [hunk 1]
```diff
@@ -417,14 +417,19 @@ def _handle_exception(future: asyncio.Future):
...
+            "block_height": self.__block_manager.blockchain.block_height,
+            "event": subscribe_event
+        }
         # try websocket connection, and handle exception in callback
-        asyncio.ensure_future(self.__node_subscriber.start(
-            block_height=self.__block_manager.blockchain.block_height,
-            event=subscribe_event,
-        ), loop=MessageQueueService.loop).add_done_callback(_handle_exception)
+        task = asyncio.ensure_future(
+            self.__node_subscriber.start(**subscribe_params),
```
Comment (metalg0su) [inline loopchain/channel/channel_service.py]: How about pass `block_height` and `event` as a keyword args, not the unpacked dict?

File: loopchain/peermanager/peer_loader.py [hunk 2]
```diff
@@ -49,13 +49,13 @@ def _load_peers_from_file(peer_manager: 'PeerManager'):
     @staticmethod
     def _load_peers_from_rest_call(peer_manager: 'PeerManager'):
         rs_client = ObjectManager().channel_service.rs_client
-        is_block_version_0_3 = PeerLoader._is_block_version_0_3(rs_client)
         crep_root_hash = conf.CHANNEL_OPTION[ChannelProperty().name].get('crep_root_hash')
 
-        if is_block_version_0_3 and crep_root_hash:
-            return PeerLoader._get_reps_by_root_hash_call(peer_manager, rs_client, crep_root_hash)
+        if not crep_root_hash:
+            logging.error(f"There's no crep_root_hash to initialize.")
```
Comment (metalg0su) [inline loopchain/peermanager/peer_loader.py]: The channel option `crep_root_hash` is not optional but required to run loopchain node, Right?

File: loopchain/peer/peer_service.py [hunk 3]
```diff
@@ -281,4 +275,4 @@ def close(self):
             await StubCollection().create_icon_score_stub(channel_name)
 
     def _load_channel_infos(self):
-        self._channel_infos = self._get_channel_infos()
+        self._channel_infos = conf.CHANNEL_OPTION
```
Comment (metalg0su) [inline loopchain/peer/peer_service.py]: Is this method needed?
How about replace `self._load_channel_infos()` as `self._channel_infos = conf.CHANNEL_OPTION`
and remove this method?

**Review by windies21 — APPROVED** [review]

File: loopchain/peer/peer_service.py (hunk 3 above)
Comment (laurenceyoon) [reply loopchain/peer/peer_service.py]: Good idea!

File: loopchain/peermanager/peer_loader.py (hunk 2 above)
Comment (laurenceyoon) [reply loopchain/peermanager/peer_loader.py]: Yep. `crep_root_hash` is required for citizen node.

File: loopchain/channel/channel_service.py (hunk 1 above)
Comment (laurenceyoon) [reply loopchain/channel/channel_service.py]: Seems reasonable. Updated!

**Review by metalg0su — APPROVED** [review]

**Review by zsaladin — APPROVED** [review]

**Pull request merged**